src/TrinaryLogic.php

```diff
@@ -47,6 +47,18 @@ public static function createFromBoolean(bool $value): self
 		return self::$registry[$yesNo] ??= new self($yesNo);
 	}
 
+	public static function createFromBooleanType(BooleanType $type): self
```

Comment:
You had the same idea, nice! But I'd actually like `BooleanType::toTrinaryLogic()` more :)

Reply:
hehe I see. done